### PR TITLE
Add new Adapter brainyBidAdapter

### PR DESCRIPTION
**Pull request by brainymisio**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'brainy',
  params: {
    accountID: 3481
    slotID: 5569
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer:support@mg.brainy-inc.co.jp
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by brainymisio** [comment]
@ndhimehta could you please review the PR and let us know?

**Comment by brainymisio** [comment]
@bretg could you please review the PR and let us know?

**Comment by brainymisio** [comment]
@ndhimehta  could you please review the PR and let us know?

File: modules/brainyBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+   * @param {object} bid the brainy bid to validate
+   * @return {boolean}
+   */
+  /**
+   * adUnits.bidに値が入っているかを判断する
+   * @param  {object} bid 検証する入札リクエスト
+   * @return {boolean}
+   */
+  isBidRequestValid: function(bid) {
+    return !!(bid && bid.params && bid.params.acountID && bid.params.slotID);
```
Comment (snapwich) [inline modules/brainyBidAdapter.js]: This looks like a typo.  should all `acountId` be `accountId`?  

Also, can you submit corresponding docs pull-request to https://github.com/prebid/prebid.github.io ?

**Comment by brainymisio** [comment]
@snapwich Thanks for the review, I believe our last commits should have solved all the issues.
We made a PR against https://github.com/prebid/prebid.github.io/tree/master/dev-docs/bidders, can you please check here: prebid/prebid.github.io#757. Thanks

**Pull request merged**